src/routing/Page.tsx

```diff
@@ -58,6 +58,15 @@ export default function Page(props: PageProps): JSX.Element {
         </div>
       )}
 
+      {previousRoute && !nextRoute && !props.hidePreviousButton && (
```

Comment:
`hidePreviousButton` is only currently used in our ConfirmationPage so that buttons don't show up.